lib/Core/TxTree.cpp

```diff
@@ -1949,6 +1952,8 @@ std::string TxTree::getInterpolationStat() {
   std::stringstream stream;
   stream << "KLEE: done: Total reduced symbolic execution tree nodes = "
 		 << TxTreeGraph::nodeCount  << "\n";
+  stream << "KLEE: done: Total number of visited basic blocks = "
+		 << TxTree::blockCount  << "\n";
```

Comment:
@rasoolmaghareh TxTree:: is unnecessary here since this function belongs to TxTree.